addons/barcodes/static/src/barcode_handlers.js

```diff
@@ -57,6 +57,7 @@ export const COMMANDS = {
 
```

Comment:
alternatively, we can edit COMMANDS and add an empty function for "cancel" and "main_menu";

Reply:
I prefere this solution like already made here https://github.com/odoo/enterprise/blob/86132aa1e7e3bbc09f440213a59ccb3e86bab808/stock_barcode/static/src/components/main.js#L22-L23